scripts/check_compliance.py

```diff
@@ -356,14 +363,17 @@ def check_no_undef_outside_kconfig(self, kconf):
         undef_to_locs = collections.defaultdict(list)
 
         # splitlines() supports various line terminators
-        for line in grep_stdout.decode("utf-8").splitlines():
-            path, lineno, sym_name = line.split("\0")
+        for grep_line in grep_stdout.decode("utf-8").splitlines():
+            path, lineno, line = grep_line.split("\0")
 
-            # [7:] removes the "CONFIG_" prefix
-            if sym_name[7:] not in defined_syms and \
-               sym_name not in UNDEF_KCONFIG_WHITELIST:
+            # Extract symbol references (might be more than one) within the
+            # line
+            for sym_name in re.findall(regex, line):
```

Comment:
Did you manually test multiple matches on the same line? Just to be on the safe side. I guess these must be rare.

Reply:
Yup, accounting for multiple matches on one line.